ports/mimxrt/boards/deploy_mimxrt.md

```diff
@@ -1,11 +1,49 @@
-Firmware can be loaded to the MIMXRT development boards in various ways. The most convenient
-one is using the built-in support MCU. When a PC is connected to the debug USB port, a drive
-icon will appear. Firmware can be uploaded to the board by copying it to this drive. The copy
-and flashing will take a few moments. At the end of the upload, the drive icon will disappear
-and reappear again. Then the reset button has to be pushed, which starts the MicroPython firmware.
+## Firmware installation options
+
+There are two ways to load the MicroPython firmware to the device:
+
+1. Load the MicroPython firmware directly to the device. The MicroPython
+firmware files for that method have the extension .bin or .hex and are available
+at the MicroPython download site.
```

Comment:
Will the .bin/.hex files work without having a bootloader installed?  I don't think they will, because `interrupts_start` and `text_start` have been changed.

Reply:
Yes. They work. I tested that by erasing the flash of a device and then installing the .bin and .hex files. At Teensy this is the default. But to be sure, I'll repeat the test.